src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -70,7 +70,7 @@ private void NotifyPropertyChanged(string propertyName)
         private double lightAzimuthDegrees = 45.0;
         private double lightElevationDegrees = 35.0;
         private int renderingTier;
-        private DynamoViewModel viewModel;
+        private IWatchViewModel viewModel;
```

Comment:
I am OK with this. But can we name this better? that is naming the viewModel to its class name.
 Like watchviewmodel.dynamoviewmodel or viewModel.dyanmoViewModel?  It is bit confusing with the name as viewmodel.ViewModel.  Otherwise LGTM.